spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/bigquery/GcpBigQueryProperties.java

```diff
@@ -36,6 +36,8 @@
 
 	private String projectId;
 
+	private String datasetName;
```

Comment:
Can you add javadocs for all of the fields? I believe the IDA with Spring plugin uses this.
Also, I think it needs to be formatted like this:
```java
	/**
	 * Overrides the GCP OAuth2 credentials specified in the Core module.
	 */
```

Reply:
Done.